common/ideas/00_basic_ideas.txt

```diff
@@ -705,350 +673,369 @@ maritime_ideas = {
+		power_projection_from_insults = 0.20
 	}
-	naval_drill = {
-		naval_morale = 0.10
+	sea_hawks = {
+		free_leader_pool = 1
+		admiral_cost = -0.25
 	}
-	copper_bottoms = {
-		naval_attrition = -0.25
+	naval_fighting_instruction = {
+		siege_blockade_progress = 1
 	}
-	massed_battery = {
-		artillery_power = 0.1
+	superior_seamanship = {
+		naval_morale = 0.1
+		heavy_ships_power = 0.20
 	}
 
 	ai_will_do = {
-		factor = 0.49
+		factor = 0.31
+		modifier = {
+			factor = 0
+			is_tribal = yes
+		}
+		modifier = {
+			factor = 0
+			primitives = yes
+		}
+		modifier = {
+			factor = 0
+			NOT = { num_of_ports = 1 }
+			NOT = { is_colonial_nation = yes }
+		}
+		modifier = {
+			factor = 0.31
+			NOT = { num_of_ports = 10 }
+		}
+		modifier = {
+			factor = 0.31
+			num_of_cities = 50
+		}
+
+		modifier = {
+			factor = 10
+			OR = {
+				has_reform = pirate_republic_reform
+				has_reform = pirate_kingdom
+				has_reform = war_against_the_world_government
+				has_reform = black_market_consortium_government
+			}
+		}
 	}
 }
 
 
-quantity_ideas = {
-	category = MIL
+influence_ideas = {
+	category = DIP
 
 	bonus = {
-		land_forcelimit_modifier  = 0.50
+		reduced_stab_impacts = yes
 	}
-	mass_army = {
-		global_manpower_modifier = 0.50
+
+	tribute_system = {
+		vassal_income = 0.25
 	}
-	the_young_can_serve = {
-		manpower_recovery_speed = 0.20
+
+	additional_loyalist_recruitment = {
+		reduced_liberty_desire = 20
 	}
-	enforced_service = {
-		global_regiment_cost = -0.1
+
+	integrated_elites = {
+		diplomatic_annexation_cost = -0.33
 	}
-	the_old_and_infirm = {
-		land_maintenance_modifier = -0.05
+	buffer_states = {
+		diplomatic_upkeep = 2
 	}
-	camp_followers = {
-		global_supply_limit_modifier = 0.33
+	diplomatic_influence = {
+		diplomatic_reputation = 3
+		accept_vassalization_reasons = 10
 	}
-	garrison_conscription = {
-		garrison_size = 0.25
+	bolster_tensions = {
+		can_fabricate_for_vassals = yes
+		rebel_support_efficiency = 0.66
 	}
-	expanded_supply_trains = {
-		land_attrition = -0.1
+	marcher_lords = {
+		vassal_forcelimit_bonus = 1
 	}
 
 	ai_will_do = {
-		factor = 1.0
+		factor = 0.5
+		modifier = {
+			factor = 1.5
+			is_emperor = yes
+		}
+		modifier = {
+			factor = 1.7
+			vassal = 1
+		}
+		modifier = {
+			factor = 1.7
+			vassal = 2
+		}
 	}
 }
 
 
-expansion_ideas = {
-	category = ADM
-	important = yes
+# MIL Ideas
+
+aristocracy_ideas = {
+	category = MIL
 
 	bonus = {
- 		min_autonomy_in_territories = -0.1
+		global_regiment_cost = -0.33
+		nobles_loyalty_modifier = 0.1
 	}
 
-	additional_colonists = {
-		colonists = 1
-	}
-	additional_merchants = {
-		merchants = 1
-	}
-	faster_colonists = {
-		global_colonial_growth = 20
+	trigger = {
+		OR = {
+			has_government_attribute = enables_aristocratic_idea_group
+			government = theocracy
+			government = tribal
+		}
 	}
-	factories = {
-		rival_border_fort_maintenance = -0.5
-		center_of_trade_upgrade_cost = -0.2
+
+	noble_knights = {
+		cavalry_power = 0.10
```

Comment:
I would bump this up perhapt to 15%, it runs the chance of just being ignored if it doesn't make cav attractive enough. Could also consider a cav cost reduction, somewhere in here, so the CCA isn't just a throwaway idea